larva/src/main/java/org/frankframework/larva/LarvaTool.java

```diff
@@ -816,75 +816,75 @@ public String initScenariosRootDirectories(String realPath, String paramScenario
 		String currentScenariosRootDirectory = null;
 		if (realPath == null) {
 			errorMessage("Could not read webapp real path");
-		} else {
-			if (!realPath.endsWith(File.separator)) {
-				realPath = realPath + File.separator;
-			}
-			Map<String, String> scenariosRoots = new HashMap<>();
-			Map<String, String> scenariosRootsBroken = new HashMap<>();
-			int j = 1;
-			String directory = appConstants.getProperty("scenariosroot" + j + ".directory");
-			String description = appConstants.getProperty("scenariosroot" + j + ".description");
-			while (directory != null) {
-				if (description == null) {
-					errorMessage("Could not find description for root directory '" + directory + "'");
-				} else if (scenariosRoots.get(description) != null) {
-					errorMessage("A root directory named '" + description + "' already exist");
-				} else {
-					String parent = realPath;
-					String m2eFileName = appConstants.getProperty("scenariosroot" + j + ".m2e.pom.properties");
-					if (m2eFileName != null) {
-						File m2eFile = new File(realPath, m2eFileName);
-						if (m2eFile.exists()) {
-							debugMessage("Read m2e pom.properties: " + m2eFileName);
-							Properties m2eProperties = readProperties(null, m2eFile, false);
-							parent = m2eProperties.getProperty("m2e.projectLocation");
-							debugMessage("Use m2e parent: " + parent);
-						}
-					}
-					directory = getAbsolutePath(parent, directory, true);
-					if (new File(directory).exists()) {
-						debugMessage("directory for ["+description+"] exists: " + directory);
-						scenariosRoots.put(description, directory);
-					} else {
-						debugMessage("directory ["+directory+"] for ["+description+"] does not exist, parent ["+parent+"]");
-						scenariosRootsBroken.put(description, directory);
+			return null;
+		}
```

Comment:
No functional change in this method; just an early return.

Reply:
Dat zie ik nog niet direct, waarom dit geen functionele change is?

Reply:
De code is lastig te lezen zo dus daarom kan ik dat niet zo goed beoordelen. 😦

Reply:
OK ik zie het nu in de IDE duidelijker!

Reply:
Ja, ik dacht 'ik zet het er bij', zodat het makkelijker te beoordelen is 😉